specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/MultivariateAnomalyDetector.json

```diff
@@ -0,0 +1,1192 @@
+        "errors": {
+          "type": "array",
+          "description": "Error message for the current timestamp.",
+          "items": {
+            "$ref": "#/definitions/ErrorResponse"
+          }
+        }
+      },
+      "type": "object"
+    },
+    "AnomalyValue": {
+      "required": [
+        "isAnomaly",
+        "score",
+        "severity"
+      ],
+      "properties": {
+        "isAnomaly": {
+          "type": "boolean",
+          "description": "True if an anomaly is detected at the current timestamp.",
+          "example": true
+        },
+        "severity": {
+          "type": "number",
+          "description": "Indicates the significance of the anomaly. The higher the severity, the more significant the anomaly is.",
+          "example": 0.8,
+          "minimum": 0.0,
+          "maximum": 1.0
+        },
+        "score": {
+          "type": "number",
+          "description": "Raw anomaly score of severity, will help indicate the degree of abnormality as well.",
+          "example": 0.3,
+          "minimum": 0.0,
+          "maximum": 2.0
+        },
+        "interpretation": {
+          "type": "array",
+          "items": {
+            "description": "Interpretation contains more details of the anomaly, which will help with root cause analysis.",
+            "allOf": [
+              {
+                "$ref": "#/definitions/AnomalyInterpretation"
+              }
+            ]
+          }
+        }
+      },
+      "type": "object"
+    },
+    "AnomalyInterpretation": {
+      "properties": {
+        "variable": {
+          "type": "string",
+          "description": "Variable.",
+          "example": "variable_1"
+        },
+        "contributionScore": {
+          "type": "number",
+          "description": "This score shows the percentage contributing to the anomalous timestamp. A number between 0 and 1.",
+          "example": [
+            0.3324159383
+          ]
+        },
+        "correlationChanges": {
+          "example": {
+            "changedVariables": [
+              "variable_2",
+              "variable_3"
+            ]
+          },
+          "$ref": "#/definitions/CorrelationChanges"
+        }
+      },
+      "type": "object"
+    },
+    "CorrelationChanges": {
+      "properties": {
+        "changedVariables": {
+          "type": "array",
+          "description": "The correlated variables that have correlation changes under an anomaly.",
+          "example": [
+            "variable_2",
+            "variable_3"
+          ],
+          "items": {
+            "type": "string"
+          }
+        }
+      },
+      "type": "object"
+    },
+    "LastDetectionRequest": {
+      "required": [
+        "topContributorCount",
+        "variables"
+      ],
+      "properties": {
+        "variables": {
+          "type": "array",
+          "description": "Variables",
+          "example": [
+            {
+              "variableName": "Variable_1",
```

Comment:
This example does not not conform to the `VariableValues` schema because it is missing the required `name` property. Should "variableName" be just "name"?

Reply:
Change all "variableName"s to "variable"s.